CHANGELOG.md

```diff
@@ -1,12 +1,19 @@
 ## next
 
+## 0.31
+
 *Important!*
 - The next release will be 1.0.0, which means that master will contain breaking changes.
 
 *Enhancements*
+- (alpha) Add a new `krane global-deploy` task for deploying global resources. ([#602](https://github.com/Shopify/kubernetes-deploy/pull/602))
```

Comment:
(alpha) Add a new `krane global-deploy` task for deploying global resources. Expect changes in pruning behavior before 1.0 release.